docker/walrus-antithesis/build-walrus-image-for-antithesis/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Similarly, can we move this to walrus/scripts?

Reply:
This script is quite particular for the antithesis docker image build. I'm not sure if it should live in scripts..? I thought about moving it to `docker/walrus-antithesis` but that directory has multiple Dockerfiles, which sort of goes against the pattern in the other `docker/*` directories. So, if anything, I'd like to make the pattern of file layouts in the `docker/*` subdirs all identical, with their corresponding `build.sh` in that `docker/*/build.sh` position, until we unify all of the build.sh scripts into something that's in `scripts/`. Does that make sense? I see it as a couple hops before we get all the way cleaned up.

Reply:
I take it back. This script is generic. Yes, we should move it and the others like it in `docker/*/build.sh` into scripts, or into `docker/`. The trouble is that there is already a pattern of using copies of this `build.sh` script over in the `sui` codebase that follows the simplified pattern discussed here. It would be good to avoid deviating from that pattern unless we plan to clean up all instances of the pattern across all codebases, imho

Reply:
@pei-mysten I think you may be the owner/originator of this pattern. I could unify all of these with a little bit of work, but I'd like to understand your thoughts on this layout/duplication of the build.sh script, etc before doing anything beyond copy/pasting.